python/revng/daemon/schema.graphql.tpl

```diff
@@ -12,6 +12,8 @@ type Query {
     {%- for rank in structure.keys() %}
     {{ rank.name }}{{ rank | rank_param }}: {{ rank.name | capitalize }}!
     {%- endfor %}
+
+    debug: Debug!
```

Comment:
What's this?

Reply:
I'm unsure where to include the `verify_{global,diff}` functions, so I put them in a separate section

Reply:
I'd put them on the side of `apply_diff`

Reply:
technically they are not mutations, so putting them next to `apply_diff` would be misleading IMHO

Reply:
Next to where fetch a global then?

Reply:
Done